src/components/ArticleGrid.jsx

```diff
@@ -28,10 +40,32 @@ export default function ArticleGrid({ topic }) {
       return newArticles;
     });
   };
+  const toggleOrder = () => {
+    setOrder((prevOrder) => (prevOrder === "asc" ? "desc" : "asc"));
+  };
+  const handleSortChange = (event) => {
+    setSortBy(event.target.value);
+  };
   if (isLoading) return <p>Loading...</p>;
   return (
     <>
       <section>
+        <Toolbar sx={{ padding: "1em" }}>
+          <Stack direction="row" spacing={2} sx={{ marginLeft: "auto" }}>
+            <Stack direction="row" alignItems="center" spacing={2}>
+              <Typography variant="h6">Sort by:</Typography>
+              <Select value={sortBy} onChange={handleSortChange}>
+                <MenuItem value="created_at">Date</MenuItem>
+                <MenuItem value="comment_count">Comments</MenuItem>
+                <MenuItem value="votes">Votes</MenuItem>
+              </Select>
+            </Stack>
+            <Stack direction="row" alignItems="center" spacing={2}>
+              <Typography variant="h6">Order:</Typography>
+              <Button onClick={toggleOrder}>{order}</Button>
+            </Stack>
+          </Stack>
+        </Toolbar>
         <Pagination
```

Comment:
As with the sorting toolbar, the pagination pips need a different colour scheme as they are not visible at the moment.